### PR TITLE
#261 Check executable exists and CFBundleExecutable is valid

### LINKED ISSUE #261: Verify bundleExecutable matches executable name
Check that executable name is the same as the bundleExecutable, or provide a more meaningful message in case of error:

```
[ 70%] VerifyingApplication
ios-deploy[3658:105136] [ !! ] Error 0xe8000067: There was an internal API error. AMDeviceSecureInstallApplication(0, device, url, options, install_callback, 0)
```

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #261

### CODE REVIEW

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4815826&bid=16015657)

File: src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java [hunk 1]
```diff
@@ -96,13 +96,24 @@ public InfoPlist(ProcessPaths paths, InternalProjectConfiguration projectConfigu
...
     public Path processInfoPlist() throws IOException {
-        Path workDir = appPath;
-        String executableName = getExecutableName(projectConfiguration.getAppName(), sourceOS);
+        String appName = projectConfiguration.getAppName();
+        String executableName = getExecutableName(appName, sourceOS);
+        Path executable = appPath.resolve(executableName);
+        if (!Files.exists(executable)) {
+            String errorMessage = "The executable " + executable + " doesn't exist.";
+            if (!appName.equals(executableName) && Files.exists(appPath.resolve(appName))) {
+                errorMessage += "\nMake sure the CFBundleExecutable key in the Default-Info.plist file is set to: " + appName;
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java]: where is that Default-Info.plist? In a src/gensrc/target dir? Make sure that if there are multiple ones (e.g. copied or generated) the user modifies the correct one.

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4815826&bid=16015657)

**Comment by eugener** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4815826&bid=16015657)

File: src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java (hunk 1 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java]: Fixed

**Pull request merged**